src/components/molecules/ExecutionDetails/TestExecutionDetails/TestExecutionDetailsTabs.tsx

```diff
@@ -24,6 +27,7 @@ const LogOutput = lazy(() => import('@molecules').then(module => ({default: modu
 
 const TestExecutionDetailsTabs: React.FC = () => {
   const {data} = useContext(ExecutionDetailsContext);
+  const {pluginSlots} = useContext(PluginsContext);
```

Comment:
It would be safer to have hooks for that, like:

```ts
const usePluginSlot = <T>(name: string, defaults?: T): T => {
  const {pluginSlots} = useContext(PluginsContext);
  return pluginSlots || defaults;
};
```